B827EBFFFE374870.json

```diff
@@ -0,0 +1,17 @@
+{
+	"gateway_conf": {
+	 	"gateway_ID": "B827EBFFFE374870",
+		"servers": [ 
+			{ "server_address": "fieldback.net",
```

Comment:
Is that really a TTN / LoRaWAN server address?

Reply:
Yes, that's our own LoRa server.